### ISSUE TITLE
Support for 3.9

### ISSUE THREAD

**Issue by wamacdonald89** [git-artes/gr-tempest#12]
Current release is incompatible with GNU Radio 3.9 due to removal of GrSWIG. Any plans on supporting version 3.9? What would be required to port this to newer versions of GNU Radio?

**Comment by dkozel** [comment]
The guide for porting from GNU Radio 3.8 to 3.9 is here:
https://wiki.gnuradio.org/index.php?title=GNU_Radio_3.9_OOT_Module_Porting_Guide

3.10 has now been released, but most out-of-tree modules do not require changes to be compatible.
https://wiki.gnuradio.org/index.php?title=GNU_Radio_3.10_OOT_Module_Porting_Guide

**Comment by bkerler** [comment]
Support for gnuradio 3.9 - 3.11 has been added via pr #16

**Comment by kalaksi** [comment]
@bkerler I'm having issues with the PR using gnuradio 3.9.5.0.
First, I changed the gnuradio package version in CMakeLists.txt from 3.10 to 3.9, but seems that there are some incompatibilities:

```
CMake Error at CMakeLists.txt:47 (include):
  include could not find load file:

    GrMinReq


CMake Error at CMakeLists.txt:53 (include):
  include could not find load file:

    GrCompilerSettings
```
The lines look similar to ones in gnuradio's CMakeLists.txt that were introduced in version 3.10.
Additionally, I got multiple errors similar to this one:
```
CMake Error at /usr/local/lib/cmake/gnuradio/GrPybind.cmake:195 (file):
  file MD5 failed to read file
  "/home/user/SDR/gr-tempest/python/tempest/bindings/../../../include/tempest/fine_sampling_synchronization.h":
  No such file or directory
Call Stack (most recent call first):
  python/tempest/bindings/CMakeLists.txt:39 (GR_PYBIND_MAKE_OOT)
```
The path should be `include/gnuradio/tempest/fine_sampling_synchronization.h`. Not sure what the root cause is since I didn't look into it at this time.

**Comment by bkerler** [comment]
Yes, some lines were added for gnuradio 3.10. I might port it specifically to 3.9 if that is really needed.
